ftml/test/anchor2-alias-special.json

```diff
@@ -0,0 +1,43 @@
+{
+    "input": "[[*anchor_ href=\"http://example.com\" style=\"color: green;\"]]My link[[/anchor]]",
+    "tree": {
+        "elements": [
+            {
+                "element": "container",
+                "data": {
+                    "type": "paragraph",
```

Comment:
Noting that Wikidot does not create a wrapping paragraph here, is the divergence intentional?

Reply:
Yeah...

So the paragraph logic is _closer_ now than it's ever been, but there are still several deficiences. The issue [FTML-73](https://scuttle.atlassian.net/browse/FTML-73) exists to ensure we iron out these issues.

For now, as long as the output is consistent with other rules, we'll treat it as valid.